### PR TITLE
feat: swap to negativo17 as nvidia driver source

### PR DESCRIPTION
**Pull request by bsherman**
This reworks all the bits in akmods for nvidia:
- removes nvidia driver version as only latest is supported
- adds negativo17-fedora-nvidia repo to ublue-os-nvidia-addons RPM
- removes extra metadata from the nvidia-vars create in image

This PR will require a corresponding PR in `hwe` to ensure proper repos are being used and the new package names are installed since negativo17 has some different package naming.

### CODE REVIEW

**Review by KyleGospo — APPROVED** [review]

**Review by noelmiller — APPROVED** [review]

**Comment by bsherman** [comment]
Despite initially clean builds which looked good (and tested good locally) when we got approvals, it seems like the timing matched up with a kernel update which is causing version skew with rpm repos.

Probably wait until Sunday afternoon to try merging again.

**Pull request merged**
